### ISSUE TITLE
MPRIS Control does not work in KDE

### ISSUE THREAD

**Issue by archisman-panigrahi** [rafaelmardojai/blanket#96]
The play/pause control does not work in KDE Plasma (KDE Neon 20.04 and Kubuntu 20.04) with the laptop's function keys (play/pause works when I click on the play/pause button with mouse).
However, the MPRIS control does work in the same laptop in GTK based DEs, like Cinnamon and XFCE. I used the latest git branch.
Also, it does not show up in KDE sound applet as an application playing music.

Most GTK APIs work very well in KDE Plasma nowadays.
In case this can be implemented in KDE with minor modification to the code, it would be very convenient.

**Comment by rafaelmardojai** [comment]
Others GNOME sound apps like Lollypop work as expected?

**Comment by apandada1** [comment]
Yes, Lollypop works as expected (Tried v1.2.35 in the Ubuntu 20.04 repositories). When I play something, Lollypop also registers itself as a media player in the KDE notifications, but Blanket does not.

![Screenshot_20210428_194918](https://user-images.githubusercontent.com/3063132/116420335-844c1100-a85b-11eb-90ee-9536aa6b828c.png)

<sup> Lollypop in KDE's media player notifications </sup>

**Comment by rafaelmardojai** [comment]
Have you seen if Plasma logs some error message?

**Comment by apandada1** [comment]
I don't know where such error messages (if they exist) are stored. Blanket does not print anything in the terminal.

**Comment by rafaelmardojai** [comment]
> I don't know where such error messages (if they exist) are stored. Blanket does not print anything in the terminal.

Maybe with `journalctl`?

**Comment by apandada1** [comment]
`journalctl -r` has too many entries. How do I find the (possible) error log of blanket in that?

**Comment by rafaelmardojai** [comment]
Maybe only looking for KDE entries? `journalctl -f | grep --line-buffered kde`

**Comment by hydrargyrum** [comment]
blanket does not seem to support mpris with https://github.com/altdesktop/playerctl either
